tox.ini

```diff
@@ -1,26 +1,14 @@
 [tox]
 distshare = {homedir}/.tox/distshare
-envlist = py27-pytest{36,37,38,39,310,4,41,42,43,44,45,46},
-          py37-pytest{36,37,38,39,310,4,41,42,43,44,45,46,5,latest},
+envlist = py27-pytest4,
```

Comment:
I'd like to keep testing against all the pytest >= 4.3

Reply:
Updated! And remembered to update `install_requires` this time too.